packages/ui-extensions/src/surfaces/admin/components/Heading/examples/basic-heading.html

```diff
@@ -0,0 +1,3 @@
+<s-box background="surface-subdued" padding="base" borderRadius="base">
```

Comment:
I'm not sure what `surface-subdued` is - is that a real background? TypeScript doesn't seem to think so.

Also, any reason we want these headings in `s-box` to begin with? Seems we could have the same examples, but not inside these boxes.

Reply:
corrected all tokens wrong in all docs, plus removed unnecessary boxes